app/services/tracking.js

```diff
@@ -135,10 +134,10 @@ export default class TrackingService extends Service {
 
       let task = "Unknown Task";
 
-      if (get(this, "activity.task.content")) {
-        const c = get(this, "activity.task.project.customer.name");
-        const p = get(this, "activity.task.project.name");
-        const t = get(this, "activity.task.name");
+      if (this.activity.task.content) {
+        const c = this.activity.task.get("project.customer.name");
+        const p = this.activity.get("task.project.name");
+        const t = this.activity.get("task.name");
```

Comment:
This seems a little inconsistent. Why don't we need the get for the first `task` access?

Reply:
it was throwing errors without it